src/setup/multiplayer.c

```diff
@@ -266,6 +266,10 @@ static void StartGame(int multiplayer)
         {
             AddCmdLineParameter(exec, "-altdeath");
         }
+        else if (deathmatch == 3)
```

Comment:
Please leave another comment here indicating that this is a Crispy-specific change.